src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -226,6 +227,9 @@ def launch_fluent(
     The allocated machines and core counts are queried from the scheduler environment and
     passed to Fluent.
     """
+    if version and gpu:
+        if version != "3d" and gpu:
```

Comment:
Why two ifs like this? Why not just the second one?

Reply:
@seanpearsonuk I thought explicit string based comparison should be checked only if both version and gpu is given. Anyway I updated it now. Thank you.